components/functions-library-mgt/org.wso2.carbon.identity.functions.library.mgt/src/main/java/org/wso2/carbon/identity/functions/library/mgt/dao/impl/FunctionLibraryDAOImpl.java

```diff
@@ -56,7 +59,7 @@
      * @throws FunctionLibraryManagementException
      */
     public void createFunctionLibrary(FunctionLibrary functionLibrary, String tenantDomain)
-            throws FunctionLibraryManagementException {
+            throws FunctionLibraryManagementServerException {
```

Comment:
why this change is required? isn't FunctionLibraryManagementServerException a subclass of FunctionLibraryManagementException?

Reply:
[Fixed commit](https://github.com/wso2/carbon-identity-framework/pull/2679/commits/9a526929402c0b6fb5ef8eab5d87a89fbaeee7c8)